packages/pyast/pyast.0.2.0/opam

```diff
@@ -0,0 +1,45 @@
+opam-version: "2.0"
+synopsis: "Python AST"
+description: """
+This library provides versioned abstract syntax tree for all Python
+versions from Python 2.5 to Python 3.11.
+"""
+maintainer: ["Thierry Martinez <thierry.martinez@inria.fr>"]
+authors: ["Thierry Martinez <thierry.martinez@inria.fr>"]
+license: "BSD-2-Clause"
+homepage: "https://github.com/thierry.martinez/pyast"
+doc: "https://github.com/thierry.martinez/pyast"
+bug-reports: "https://github.com/thierry.martinez/pyast"
+depends: [
+  "ocaml" {>= "4.03.0"}
+  "dune" {>= "2.8"}
+  "pyml" {>= "20210226"}
+  "refl" {>= "0.4.0"}
+  "cmdliner" {>= "1.0.4"}
```

Comment:
```suggestion
  "cmdliner" {>= "1.1.0"}
```
Since it uses the new `Cmdliner.Cmd` module

Reply:
Thank you very much for having fixed this. Ported upstream: https://github.com/thierry-martinez/pyast/commit/4b6cecc3cdfea6932567c38b866e5c4017f11fc7